.github/workflows/tests.yml

```diff
@@ -0,0 +1,352 @@
+name: 'Tests'
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+
```

Comment:
Since we're using the same node version for each job, we could use a root-level shared env like the following:
``` yaml
env:
  NODE_VERSIONS: [12, 14]
```

and then use
``` yaml
matrix:
    node: $NODE_VERSIONS
```
in each job

It would be easier/more convenient to manage for future updates.
What do you think?

Reply:
Référence: [here](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#env)

Reply:
We actually cannot access the env in the matrix key as per the doc the env access is pretty limited https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#env-context

Reply:
:disappointed: